keyboards/decent/tkl/keyboard.json

```diff
@@ -0,0 +1,302 @@
+{
+    "manufacturer": "Bertrand Le roy",
+    "keyboard_name": "DecenTKL",
+    "maintainer": "bleroy",
+    "url": "https://github.com/bleroy/3d-junkyard/blob/main/DecenTKL/",
+    "tags": ["ansi", "tkl", "rgb", "oled", "encoder", "atari"],
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "features": {
+        "encoder": true,
+        "bootmagic": true,
+        "rgb_matrix": true,
+        "oled": true,
```

Comment:
OLED Being enabled at the keyboard level, but the drawing code existing at the keymap level usually gets flagged with the suggestion to migrate it to a keyboards/decent/tkl/tkl.c file with the appropriate refactoring from _user to _kb functions.

Reply:
Drawing code moved to tkl.cs at the keyboard level.